docs/concepts/cluster_queue.md

```diff
@@ -334,4 +250,5 @@ borrow up to the sum of min quotas from all the ClusterQueues in the cohort.
 
 ## What's next?
 
+- Create [local queues](/docs/concepts/local_queue.md)
```

Comment:
I would also add a link to Create resource flavors.
Admins might choose to create flavors and cluster queues in any order. Better cross link.